### DISCUSSION TITLE
Message for all users

### DISCUSSION THREAD

**Discussion by MaximMGTTRN** [bukhalo/nestjs-telegraf#1071] [Q&A]
Hi everyone, i have a problem... 
i try to send message for all users
i have db where i save everyone who use '/start' 
and now i try to send message from admin to someone or every person who use my bot 
but i'm not understand how i can reply in current chat 
i try ctx.sendMessage or ctx.reply but i'm not understand where i need to write userid...

**Comment by margazun** [comment]
try ctx.telegram.SendMessage(chatId, text)
